onnxruntime/core/providers/nnapi/nnapi_builtin/builders/helper.cc

```diff
@@ -368,14 +389,62 @@ bool IsValidSupportedNodesVec(const std::vector<size_t>& supported_node_vec, con
   return true;
 }
 
+// We support some operators running using uint8 internally
+// These nodes cannot use a graph input as input since onnx graph input does not carry scale/zero point info
+bool IsInternalQuantizationSupported(const Node& node, const GraphViewer& graph_viewer,
+                                     const std::vector<size_t>& group) {
+  const auto& node_type = node.OpType();
+
+  // For now only these [non-qlinear]operators will handle uint8 inputs
+  if (node_type != "Transpose" &&
+      node_type != "Resize" &&
+      node_type != "Concat" &&
+      node_type != "MaxPool") {
+    return true;
+  }
+
+  int32_t input_type;
+  if (!GetType(*node.InputDefs()[0], input_type))
+    return false;
+
+  // Not running using quantized input
+  if (input_type == ONNX_NAMESPACE::TensorProto_DataType_FLOAT)
+    return true;
+
+  if (group.empty()) {  // This is the 1st node in the group (partition)
+    LOGS_DEFAULT(WARNING) << "Node [" << node.Name() << "] type " << node_type
```

Comment:
Not quite following this. Won't the group always be empty on the first call, so supported will always be false?

Reply:
Yes, but want to log a verbose message such that we can know the reason why the node is not supported, it may not be obvious from the logging